### PR TITLE
remove a few warnings from linux build

### PR DESCRIPTION
**Pull request by AndrewMeadows**

### CODE REVIEW

File: libraries/display-plugins/src/display-plugins/OpenGLDisplayPlugin.cpp [hunk 1]
```diff
@@ -105,6 +105,8 @@ bool OpenGLDisplayPlugin::eventFilter(QObject* receiver, QEvent* event) {
             return true;
         }
         break;
+    default:
+        break;
```
Comment (AndrewMeadows) [inline libraries/display-plugins/src/display-plugins/OpenGLDisplayPlugin.cpp]: This removes a rather verbose warning about all the unhandled enum values in the switch statement.

File: libraries/gpu/src/gpu/GLBackendTransform.cpp [hunk 2]
```diff
@@ -41,7 +41,6 @@ void GLBackend::do_setViewportTransform(Batch& batch, uint32 paramOffset) {
         if (_stereo._pass) {
             vp.x += vp.z;
         }
-        int i = 0;
     } 
 
     glViewport(vp.x, vp.y, vp.z, vp.w);
```
Comment (AndrewMeadows) [inline libraries/gpu/src/gpu/GLBackendTransform.cpp]: This was unused, however I've seen some "extra" lines of code in teh render pipeline that are used as convenient hooks for setting breakpoints.  If this is one of those just let me know and we can leave it in.  (BTW, please label such debug hooks with comments, kthx).

File: tests/entities/src/main.cpp [hunk 3]
```diff
@@ -93,8 +93,8 @@ template <typename T>
 void testByteCountCoded() {
     testByteCountCodedStable<T>(0);
     testByteCountCodedStable<T>(1);
-    testByteCountCodedStable<T>(1 << 16);
-    testByteCountCodedStable<T>(std::numeric_limits<T>::max() >> 16);
+    testByteCountCodedStable<T>(1 << 8*sizeof(T));
+    testByteCountCodedStable<T>(std::numeric_limits<T>::max() >> 8*sizeof(T));
     testByteCountCodedStable<T>(std::numeric_limits<T>::max() >> 8);
     testByteCountCodedStable<T>(std::numeric_limits<T>::max() >> 1);
     testByteCountCodedStable<T>(std::numeric_limits<T>::max());
```
Comment (AndrewMeadows) [inline tests/entities/src/main.cpp]: This code is in a templated method and one of its instances was using a **quint8** for **T** which doesn't have 16 bits to shift, hence a warning about shifting off the end of the value.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 614fad08116d976de20d2948a5b5300d83603107 available at: https://gist.github.com/3fd707b1bcdf009f3cee<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5676.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5676.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: fe2d03fd3716e62f26a94f97f3f60a640e8f5630 available at: https://gist.github.com/3fd707b1bcdf009f3cee<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5676.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5676.dmg

**Pull request merged**